index.js

```diff
@@ -14,14 +15,20 @@ async function run() {
     const apiKey = core.getInput('apiKey') || process.env.CONTENT_API_KEY;
     const chapterApiURL = core.getInput('chapterApiURL') || process.env.CONTENT_CHAPTER_API_URL;
     const versionApiURL = core.getInput('versionApiURL') || process.env.CONTENT_VERSION_API_URL;
+    const mergedAt = core.getInput('mergedAt') || Date.parse(new Date());
```

Comment:
um `Date.now()` não resolve?

```suggestion
    const mergedAt = core.getInput('mergedAt') || Date.now();
```

Reply:
O `Date.now()` não vem no formato unix que o endpoint espera, ao por isso o `Date.Parse()`